calculatorApi1.js

```diff
@@ -0,0 +1,112 @@
+const express = require("express");
+const cors = require("cors")
+const app = express();
+const port = 8000;
+app.use(cors())
+
+
+const add = (num1, num2) => num1 + num2
+const subtract = (num1, num2) => num1 - num2
+const multiply = (num1, num2) => num1 * num2
+const divide = (num1, num2) => num1 / num2
+
+app.get("/", (req, res) => {
+    res.send("You made request")
+})
+
+app.get("/add/:num1/:num2", (req, res) => {
```

Comment:
This looks good, but try to turn the params into numbers before using the function.